internal/serverapi/serverapi.go

```diff
@@ -294,4 +294,5 @@ type UIPreferences struct {
 	Theme                  string `json:"theme"`                  // Specifies the theme used by the UI
 	FontSize               string `json:"fontSize"`               // Specifies the font size used by the UI
 	PageSize               int    `json:"pageSize"`               // A page size; the actual possible values will only be provided by the frontend
+	Language               string `json:"language"`               // Specifies the language used by the UI
```

Comment:
do we want to apply some default when returning the value or is empty string going to be enough?

Reply:
It is handled by KopiaUI currently. If the value is empty it defaults to "en". However, we can think about setting a default here.